public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -329,6 +330,22 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     return this.state.viewPanelScene ?? this.state.body;
   }
 
+  public getVizPanels(): VizPanel[] {
+    const panels: VizPanel[] = [];
+
+    this.state.body.forEachChild((child) => {
+      if (child instanceof SceneGridItem) {
+        const panel = child.state.body;
+
+        if (panel instanceof VizPanel) {
+          panels.push(panel);
+        }
+      }
+    });
+
+    return panels;
+  }
```

Comment:
This implementation isn't fully correct. SceneGridItems can also be part a child of SceneGridRow. Please have a look in the `transformSaveModelToScene` how the rows are created.

Also - can you please move this function from DashboardScene to `dashboardSceneGraph` util?